plugins/inputs/aliyuncms/aliyuncms.go

```diff
@@ -23,84 +23,92 @@ import (
+	  ## 5) RSA keypair credential
+	  ## 6) Environment variables credential
+	  ## 7) Instance metadata credential
+	  
+	  # access_key_id = ""
+	  # access_key_secret = ""
+	  # access_key_sts_token = ""
+	  # role_arn = ""
+	  # role_session_name = ""
+	  # private_key = ""
+	  # public_key_id = ""
+	  # role_name = ""
+	
+	  ## Specify the ali cloud region list to be queried for metrics and objects discovery
+	  ## If not set, all supported regions (see below) would be covered, it can provide a significant load on API, so the recommendation here 
+	  ## is to limit the list as much as possible. Allowed values: https://www.alibabacloud.com/help/zh/doc-detail/40654.htm
+	  ## Default supported regions are:
+	  ## 21 items: cn-qingdao,cn-beijing,cn-zhangjiakou,cn-huhehaote,cn-hangzhou,cn-shanghai,cn-shenzhen,
+	  ##           cn-heyuan,cn-chengdu,cn-hongkong,ap-southeast-1,ap-southeast-2,ap-southeast-3,ap-southeast-5,
+	  ##           ap-south-1,ap-northeast-1,us-west-1,us-east-1,eu-central-1,eu-west-1,me-east-1
+	  ##
+	  ## From discovery perspective it set the scope for object discovery, the discovered info can be used to enrich
+	  ## the metrics with objects attributes/tags. Discovery is supported not for all projects (if not supported, then 
+	  ## it will be reported on the start - for example for 'acs_cdn' project:
+	  ## 'E! [inputs.aliyuncms] Discovery tool is not activated: no discovery support for project "acs_cdn"' )
+	  ## Currently, discovery supported for the following projects:
+	  ## - acs_ecs_dashboard
+	  ## - acs_rds_dashboard
+	  ## - acs_slb_dashboard
+	  ## - acs_vpc_eip   
+	  regions = ["cn-hongkong"]
+	
+	  # The minimum period for AliyunCMS metrics is 1 minute (60s). However not all
+	  # metrics are made available to the 1 minute period. Some are collected at
+	  # 3 minute, 5 minute, or larger intervals.
+	  # See: https://help.aliyun.com/document_detail/51936.html?spm=a2c4g.11186623.2.18.2bc1750eeOw1Pv
+	  # Note that if a period is configured that is smaller than the minimum for a
+	  # particular metric, that metric will not be returned by the Aliyun OpenAPI
+	  # and will not be collected by Telegraf.
+	  #
+	  ## Requested AliyunCMS aggregation Period (required - must be a multiple of 60s)
+	  period = "5m"
+	  
+	  ## Collection Delay (required - must account for metrics availability via AliyunCMS API)
+	  delay = "1m"
+	  
+	  ## Recommended: use metric 'interval' that is a multiple of 'period' to avoid
+	  ## gaps or overlap in pulled data
+	  interval = "5m"
+	  
+	  ## Metric Statistic Project (required)
+	  project = "acs_slb_dashboard"
+	  
+	  ## Maximum requests per second, default value is 200
+	  ratelimit = 200
+	  
+	  ## Deprecated, use regions instead
+	  #discovery_regions = ["cn-hongkong"]
+	  
+	  ## how often the discovery API call executed (default 1m)
+	  #discovery_interval = "1m"
+	  
+	  ## Metrics to Pull (Required)
+	  [[inputs.aliyuncms.metrics]]
+	  ## Metrics names to be requested, 
```

Comment:
The `sampleConfig` content should start with two-spaces to generate a consistent config file. Can you please fix this formatting issue!?

Reply:
Fixed

Reply:
Oh I think my comment was confusing. You really have to have two spaces from the line start as the initial indentation is also used for the string. You can see the difference when generating a config file with `/telegraf --input-filter aliyuncms --aggregator-filter " " --processor-filter " "  --output-filter file config > aliyun.conf`.

Reply:
Forgot to push changes, now should be fixed